<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请选择您感兴趣的游戏(最多选择四个)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>跳过</string>
   </property>
   <property name="icon">
    <iconset resource="picture.qrc">
     <normaloff>:/icons/skip.png</normaloff>:/icons/skip.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/Mc.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/PUBG.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>445</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/apex.1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/computer.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>61</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/lol.png);
border-image: url(:/icons/lol.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>124</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/Three.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>445</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/pvp.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>51</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/wukong.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>45</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/Wa.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>130</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/CSGO.png);
border-image: url(:/icons/cs);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>290</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/wallpaper_engine.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>450</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/shoot.png);
border-image: url(:/icons/-zshejilei.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>630</y>
     <width>145</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>176</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>三角洲行动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>490</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>单机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>333</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>英雄联盟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>490</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>333</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wallpaper</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>333</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>黑神话悟空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>333</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>绝地求生</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>176</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CS2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>176</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我的世界</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>176</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>无畏契约</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>490</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>moba</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>490</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apex</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
